docs/forms/FORM_ERRORS.md

```diff
@@ -45,9 +26,8 @@ export interface FormError {
 }
 ```
 
-- __ElementError__
-This object contains all the errors associated to the html element.
-The identifier __htmlElementId__ can be used as an anchor link to focus on the html element on which the validation has failed
+- The `ElementError` object contains all the errors associated to the HTML element.
```

Comment:
Make sure to introduce the ElementError if you remove the code above. If not, please ignore this message :)